sklearn/cluster/bicluster.py

```diff
@@ -35,10 +35,19 @@ def _scale_normalize(X):
 
     """
     X = make_nonnegative(X)
-    row_diag = np.asarray(1.0 / np.sqrt(X.sum(axis=1))).squeeze()
-    col_diag = np.asarray(1.0 / np.sqrt(X.sum(axis=0))).squeeze()
+    row_diag = np.asarray(1.0 / np.sqrt(X.sum(axis=1)))
+    if row_diag.shape[0] != 1:
```

Comment:
Hey. are you still working on this? I'd like to get this into 0.12.1, which I'm planning to do over the weekend.